#188 Style the volunteers into a grid

isaabutaa opened the pull request:
I tried my best to match the Figma wireframe for the volunteers page. For the life of me, I could not get the social media icons to be a perfect circle. 😫 I added a very light border to the volunteer `figcaption` so it was easier to see.

Below are some screenshots of what the volunteers page looks like 375, 768, and 1230px wide.

![volunteers_375](https://user-images.githubusercontent.com/56979810/161401867-783d1704-021e-4a47-aa63-f451aa7a75c6.png)
![volunteers_768](https://user-images.githubusercontent.com/56979810/161401872-2a366731-c76d-4fb4-9e27-59ef6536a324.png)
![volunteers_1230](https://user-images.githubusercontent.com/56979810/161401878-89b922a4-a5b2-448a-a430-49fbbf0a0163.png)

isaabutaa commented on pages/assets/styles.css:
```diff
@@ -3,9 +3,10 @@
 
 :root {
-  --page-width: 800px;
+  --page-width: 1146px;
```
I increased the page width here because it was affecting the view of the volunteers page. This does, however, slightly impact the alignment of the navigation at the top of the page.

isaabutaa commented on pages/assets/volunteers.css:
```diff
@@ -0,0 +1,82 @@
+  width: 40px;
+  border-radius: 50%;
+}
+.volunteer__social-item img {
```
This is where I tried making the social media icons a circle, but for some reason I couldn't figure out, it didn't work. It only rounded the corners. I tried adding a border radius and width/height for the parent anchor tag as well, but that didn't do anything.

mxmason replied:
This is happening because the icons themselves have a transparent border, not because you're doing anything wrong! We'll have to replace these icons with something else to make them look right, which I'm not against. 

I made a codepen to hopefully illustrate how the icon itself is the problem: https://codepen.io/mxmason/pen/WNdXyVL

isaabutaa replied:
Ahh okay that makes sense. Yeah I added a border around the icon locally just to see, and you can see the circle. That's interesting

stacietaylorcima replied:
@isaabutaa Feel free to grab new icons from the internet! I get icons over here a lot: https://www.flaticon.com/premium-icon/facebook_2168281?term=social%20media&page=1&position=6&page=1&position=6&related_id=2168281&origin=search

isaabutaa commented on pages/volunteers.html:
```diff
@@ -45,37 +45,34 @@
       <img
         alt=""
         class="volunteer__photo"
-        height="400"
```
Added these styles in `volunteers.css`

stacietaylorcima reviewed (changes requested):
This is looking fantastic and so near merging to main! I have 3 requests: 

1. Let's just get new social media icons from [over here](https://www.flaticon.com/premium-icon/facebook_2168281?term=social%20media&page=1&position=6&page=1&position=6&related_id=2168281&origin=search) to replace those ones that are causing a fuss! 
3. Can we put a `width:100%` on the `body` to account for some misalignement in mobile? ([see GIF here](https://cdn.zappy.app/ec17506794453284a7b1f378686509a1.gif))
4. Let's delete the board and founder info off `Developers` page. ([screenshot](https://cdn.zappy.app/ea49eede062598d01df8a13283e49aa9.png))

isaabutaa commented:
@stacietaylorcima I implemented the changes you requested. I was a little confused about the width on the body because locally, it looked like adding the width made it less aligned, but it's possible that is because of my local environment I'm not sure. But I added the `100%` width to the body for that media query in `styles.css`. Let me know if you'd like any other changes!
